### DISCUSSION TITLE
How to give custom font-size, line-height, font-weight and letter-spacing

### DISCUSSION THREAD

**Discussion by vikas-growisto** [fastn-stack/fastn#578] [Q&A] [answered]
In the cheatsheet, we have pre-defined values for font-size and font-weight. But how can we give custom values for an individual property?
Example:

```ftd
-- ftd.text: Welcome to FTD
```

I have a text and want to set font-size to 40px and font-weight to 700. then how can we fix this?

**Comment by Arpita-Jaiswal** [comment, marked as answer]
You can define your own font type:

```ftd
-- ftd.type dtype:
size.px: 40
weight: 700
font-family: cursive
line-height.px: 65
letter-spacing.px: 5


-- ftd.text: Hello World
role: $dtype
```

In case, if you want to define a responsive font type, you can do the following:

```ftd
-- ftd.type dtype:
size.px: 40
weight: 700
font-family: cursive
line-height.px: 65
letter-spacing.px: 5

-- ftd.type mtype:
size.px: 20
weight: 100
font-family: fantasy
line-height.px: 35
letter-spacing.px: 3

-- ftd.responsive-type rtype:
desktop: $dtype
mobile: $mtype

-- ftd.text: Hello World
role: $rtype
```

**Reply by vikas-growisto** [reply]
Thanks for the solution.
Adding to this question, can you give an example of how to add a custom font-family of google fonts like [Poppins](https://fonts.google.com/specimen/Poppins) ?

**Reply by Arpita-Jaiswal** [reply]
@vikas-growisto Please check out [this discussion](https://github.com/ftd-lang/ftd/discussions/580#discussioncomment-4901254) and mark my comment as an answer if you find it useful. Also please let me know if you still have any issues.
